sdk/scheduler/src/test/java/com/mesosphere/sdk/config/validate/VerifyHostVolumePathTest.java

```diff
@@ -0,0 +1,90 @@
+package com.mesosphere.sdk.config.validate;
+
+import com.mesosphere.sdk.specification.DefaultHostVolumeSpec;
+import org.junit.Test;
+
+public class VerifyHostVolumePathTest {
+
+    /* Exception.class is generic (in case we add/change fields)
```

Comment:
IMO it'd make sense to use more granular types in these tests

Reply:
The secrets test uses just Exception anyway, which is what I copied the model from. I could change both in a later commit?